cmake/Modules/Packages/GPU.cmake

```diff
@@ -219,8 +219,15 @@ elseif(GPU_API STREQUAL "HIP")
           set(HIP_PATH $ENV{HIP_PATH} CACHE PATH "Path to which HIP has been installed")
       endif()
   endif()
-  set(CMAKE_MODULE_PATH "${HIP_PATH}/cmake" ${CMAKE_MODULE_PATH})
-  find_package(HIP REQUIRED)
+  if(NOT DEFINED ROCM_PATH)
+      if(NOT DEFINED ENV{ROCM_PATH})
+          set(ROCM_PATH "/opt/rocm" CACHE PATH "Path to which ROCM has been installed")
```

Comment:
This is an odd phrase. How about "Path where ROCm is installed" or "Path to ROCm installation"?